gulpfile.js

```diff
@@ -726,25 +714,20 @@ async function checkPackages({ skipSecurity } = {}) {
 				break;
 
 			case 'npm info':
-				return new Promise(resolve => {
-					npm.commands.view([ pkg ], true, (err, info) => {
-						if (err) {
-							log(yellow(`npm view failed for ${pkg}@${version}: ${err.message}`));
-						} else {
-							const latest = Object.keys(info)[0];
-							const next = info[latest]['dist-tags'] && info[latest]['dist-tags'].next || null;
-							dependencies[pkg].latest          = latest;
-							dependencies[pkg].latestTimestamp = info[latest].time[latest] || null;
-							dependencies[pkg].next            = next;
-							dependencies[pkg].nextTimestamp   = next && info[latest].time[next] || null;
-							dependencies[pkg].deprecated      = info[latest].deprecated;
```

Comment:
libnpm doesn't return whether the package is deprecated?

Reply:
Heh, I originally made this change ages ago, and must have missed that you'd added that. Will check and update

Reply:
Fixed!